test/integration/implementations/mongo_bulk_quotes_repo.test.ts

```diff
@@ -50,11 +50,13 @@ let mongoBulkQuotesRepo : IBulkQuoteRepo;
 
 let mongoClient: MongoClient;
 let collection : Collection;
-const connectionString = `${CONNECTION_STRING}/${DB_NAME}`;
+const connectionString = `${CONNECTION_STRING}`;
 
 describe("Implementations - Mongo Bulk Quotes Repo Integration tests", () => {
 
     beforeAll(async () => {
+        jest.setTimeout(10000);
```

Comment:
This is a good point, we shouldn't need this sort of timeout, but if we do, that should be before the `describe`.

Reply:
Either that or, if it's a problem of resource sharing, try to with the `[--runInBand](https://jestjs.io/docs/cli#--runinband)` option from jest so that only one test file runs at a time.

Reply:
That was used for debugging while connecting to remote mongo db. I just removed it. Thanks for pointing out Rui and Pedro.